sdk/trace/src/main/java/io/opentelemetry/sdk/trace/RecordEventsReadableSpan.java

```diff
@@ -190,6 +190,14 @@ public SpanData toSpanData() {
     }
   }
 
+  @Override
+  @Nullable
+  public <T> T getAttribute(AttributeKey<T> key) {
```

Comment:
Except for `hasEnded` (write-once) all other getters are for read-only fields. From a theoretical perspective, I would prefer *not* to have this getter, if we can find another solution to the original problem on the issue.